openvdb_points/unittest/TestPointDataLeaf.cc

```diff
@@ -256,6 +258,61 @@ TestPointDataLeaf::testOffsets()
 
         delete leafNode;
     }
+
+    // test bulk offset replacement without activity mask update
+
+    {
+        LeafType* leafNode = new LeafType();
+
+        for (openvdb::Index i = 0; i < LeafType::SIZE; ++i) {
+            leafNode->setOffsetOn(i, 10);
+        }
+
+        std::vector<LeafType::ValueType> newOffsets;
+        newOffsets.resize(LeafType::SIZE);
+
+        leafNode->setOffsets(newOffsets, /*updateActiveStates*/false);
```

Comment:
Please take a look at my comments above, if you still prefer updateActiveStates then let me know, and I'll go ahead and make the change...